KubeArmor/config/config.go

```diff
@@ -211,6 +217,7 @@ func LoadConfig() error {
 
 	GlobalCfg.Policy = viper.GetBool(ConfigKubearmorPolicy)
 	GlobalCfg.HostPolicy = viper.GetBool(ConfigKubearmorHostPolicy)
+	GlobalCfg.ContainerPolicy = viper.GetBool(ConfigKubearmorContainerPolicy)
```

Comment:
Any particular reason for this flag? Since I see in the code we seem to be using `GlobalCfg.Policy`

Reply:
Ah right, so the original plan was to use GlobalCfg.ContainerPolicy for running KubeArmor in container mode, but GlobalCfg.Policy with -k8s=false will do the job. I will update it